config/secrets.yml

```diff
@@ -19,4 +19,4 @@ test:
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 22ec19624918c726c3071b92dd5d6310baafd5e70b1562f55d78608f4aa0ac624e7a9b4f1c44eb1a2131d4590ce80677bd2607f93f3527e72114cc67fc57d4c7
```

Comment:
I guess you are hiding this file in future versions, but maybe there is a way to delete it from github?